lib/ansible/plugins/cliconf/vyos.py

```diff
@@ -59,7 +59,13 @@ def edit_config(self, command):
             self.send_command(cmd)
 
     def get(self, command, prompt=None, answer=None, sendonly=False):
-        return self.send_command(to_bytes(command), prompt=to_bytes(prompt), answer=to_bytes(answer), sendonly=sendonly)
+        # Avoid setting either value to b'None'
+        if prompt is not None:
+            prompt = to_bytes(prompt)
+        if answer is not None:
+            answer = to_bytes(answer)
+
+        return self.send_command(to_bytes(command), prompt=prompt, answer=answer, sendonly=sendonly)
```

Comment:
Is conversion to `to_bytes`  required while parameters to `send_command()`
Command conversion is handled here in [network_cli.py file at L288](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/connection/network_cli.py#L288)
and for prompt and answer it is done at [L305 in network_cli.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/connection/network_cli.py#L305)

Reply:
Probably not. `send()` has other issues, though, so I'll pull those in instead